src/com/oltpbenchmark/benchmarks/wikipedia/procedures/UpdatePage.java

```diff
@@ -317,7 +317,7 @@ public void execute(Connection conn, PreparedStatement p) throws SQLException{
 					successful = true;
 				} catch (SQLException esql) {
 					int errorCode = esql.getErrorCode();
-					if (errorCode == 8177)
+					if (errorCode == 8177) 
```

Comment:
Is this Cubrid specific?

Reply:
No, this code exists in the previous code. Sorry we accidentally add a white space. We will remove that.